package.json

```diff
@@ -35,32 +35,32 @@
     "url": "https://github.com/danielmoncada/date-time-picker.git"
   },
   "dependencies": {
-    "@angular/animations": "^13.3.5",
-    "@angular/common": "^13.1.2",
-    "@angular/compiler": "^13.3.5",
-    "@angular/core": "^13.1.2",
-    "@angular/forms": "^13.3.5",
-    "@angular/platform-browser": "^13.3.5",
-    "@angular/platform-browser-dynamic": "^13.3.5",
-    "@angular/router": "^13.3.5",
+    "@angular/animations": "^17.0.4",
```

Comment:
Are we sure we want to do this? The library will *not* be backward compatible if we are building with Angular 17!